core/src/foundations/ionic.utility.scss

```diff
@@ -0,0 +1,587 @@
+/**
```

Comment:
~ Non-blocking, same feedback here regarding formatting. If it isn't simple, we can come back and have prettier format these files.